src/main/java/org/broadinstitute/hellbender/tools/walkers/varianteval/VariantEvalEngine.java

```diff
@@ -485,6 +486,21 @@ public long getnProcessedLoci() {
         return traversalIntervals.stream().mapToLong(SimpleInterval::size).sum();
     }
 
+    /**
+     * @return The list of contig names to be traversed, preferentially taking user supplied intervals, but otherwise defaulting to driving variants
+     */
+    public List<String> getContigNames() {
+        final TreeSet<String> contigs = new TreeSet<>();
+        if (traversalIntervals == null) {
+            getSequenceDictionaryForDrivingVariants().getSequences().stream().map(SAMSequenceRecord::getSequenceName).forEach(contigs::add);
+        }
+        else {
+            traversalIntervals.stream().map(SimpleInterval::getContig).forEach(contigs::add);
+        }
+
+        return new ArrayList<>(contigs);
+    }
+
```

Comment:
This transformation seems pretty specific to the contig strat class, so I think it would make more sense to keep it there, and instead expose an intervals getter on `VariantEvalEngine`.

Reply:
ok